Dockerfile

```diff
@@ -16,6 +16,8 @@ WORKDIR /app
 
 # Cache breaking
 ENV BUILD_NUMBER ${BUILD_NUMBER:-1_0_0}
+ENV GIT_REF ${GIT_REF:-xxxxxxxxxxxxxxxxxxx}
```

Comment:
bake in the GIT_REF as an env var when building the image. 

This is available at this stage as seen by the old code that used it for writing the build-info json file.